src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
 // write your code here
+
+const populationElements = document.querySelectorAll('.population');
+
+const populations = Array.from(populationElements).map((pop) => {
+  return Number(pop.textContent.split(',').join(''));
+});
```

Comment:
This code block is a little bit hard to read. Consider making it more readable by breaking it down into smaller, well-named functions. For example, you could create a separate function for converting the population text content into a number.